pip-wheel-metadata/aws_sam_translator.dist-info/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why we added license file and couple of more to the repo? Was adding `pip-wheel-metada` intended?

Reply:
Fixed it. It was side-effect of `git add .`